package.json

```diff
@@ -16,7 +16,10 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "test:cypress": "cypress open",
-    "test:cypress-run": "cypress run"
+    "test:cypress-run-chrome": "cypress run --browser chrome --tag cross-browser,chrome --record --key c3eb12cb-8907-4304-b22f-07032f47b1cf",
```

Comment:
instead of putting key here you can pass it as environment variable. I think that probably better suits when we running it in container.